controllers/fg_handler.go

```diff
@@ -12,41 +30,66 @@ const (
 	Disabled
 )
 
+// This method returns a new FeatureGateStatus object
+// that contains the status of the feature gates
+// defined in the ConfigMap in the namespace
+// Return default values if the ConfigMap is not found.
+// Return values from the ConfigMap if the ConfigMap is not found. Use default values for missing entries in the ConfigMap.
+// Return an error for any other reason, such as an API error.
+func NewFeatureGateStatus(client client.Client) (*FeatureGateStatus, error) {
```

Comment:
since this method is now part of controllers package can it be made part of the KataConfigOpenShiftReconciler struct ?

Reply:
Sure thing. Can you help me understand why this is desirable?

Reply:
We can use the client and other variables that are part of the KataConfigOpenShiftReconciler struct if needed.

Reply:
Addressed with [aaeaea6](https://github.com/openshift/sandboxed-containers-operator/pull/417/commits/aaeaea6dd3e8638b9a15775b6f2cb65025214f06).